testsuite/domain/src/test/java/org/jboss/as/test/integration/domain/AbstractConfigurationChangesTestCase.java

```diff
@@ -89,7 +89,7 @@ public static void setupDomain() throws Exception {
 
     @AfterClass
     public static void tearDownDomain() throws Exception {
-        testSupport.stop();
```

Comment:
Is there still usage of the `stop()` method after your refactoring?
Should the method visibility be reduced (to protected/package/private)?

Reply:
It's still used in WildFly full.